scripts/destroy.sh

```diff
@@ -1,23 +1,28 @@
 #!/bin/bash
-NAMESPACES="flux control delivery logging monitoring security tracing istio-system"
+NAMESPACES="flux control delivery logging monitoring security istio-system"
 
-# Delete flux to stop GitOps
-kubectl delete --wait=true -R -f platform/flux/
+# Delete Flux So it doesn't re-create helmReleases
+kubectl delete --ignore-not-found --wait=true -R -f platform/flux/
 
-# Remove everything from Platform
-kubectl delete -R -f platform/
+# Delete Istio Operator and Istio Deployment
+kubectl -n istio-system get IstioControlPlane stakater-istiocontrolplane -o=json | jq '.metadata.finalizers = null' | kubectl delete -f -
 
-# Delete all namespaces
-kubectl delete namespaces $NAMESPACES
+# Delete StakaterPlatform resources
+kubectl delete --ignore-not-found --wait=true -R -f platform/
+
+# Remove HelmOperator 
+helm delete --purge helm-operator
```

Comment:
This doesn't remove the deployment against tiller ? what we are removing in `line 25 - kubectl delete deploy tiller-deploy -n kube-system`